Jenkinsfile.sdk

```diff
@@ -315,13 +315,11 @@ pipeline {
 						}
 					}
 				}
-				stage('Integation tests') {
+				stage('Integration tests') {
 					agent { node { label 'lisk-framework' }}
 					steps {
 						setup()
-						timeout(10) {
```

Comment:
any reason why we are removing `timeout`? the timeout of 10 mins was kept to ensure if the tests hang beyond a certain time limit Jenkins automatically cleans it up. 
Also, the better approach would be to add this timeout to the root level, so that any of the stages fail to finish under 10 mins will be cleaned up.

Reply:
I removed this timeout to make it that Jenkins pipeline consistent to all others. As you said this assumption for timeout can be applied to other pipeline stages as well.

Reply:
I added that option for that particular stage back. As I can't figure out to set a timeout per stage globally. Either you an set timeout for whole pipeline or individual stage separately.